src/data/markdown/docs/01 guides/06 Misc/06 Glossary.md

```diff
@@ -31,6 +29,12 @@ When discussing complex topics, it is usually a good idea to define a clear, sha
 
 An **HTTP Archive**, or **HAR file**, is a file containing logs of a browser interactions with the system under test. All of the included transactions are stored as JSON-formatted text. These archives may then be used to generate test scripts using, for instance, the [har-to-k6 Converter](https://github.com/loadimpact/har-to-k6). For more details, see the [HAR 1.2 Specification](http://www.softwareishard.com/blog/har-12-spec/).
 
+## Iteration
+
+An iteration is the same as an execution of the `default function`, or scenario `exec` function.
+
+You can either calculate iterations across all [virtual users](#virtual-users) (as done by the [Shared iterations](/using-k6/scenarios/executors/shared-iterations) executor), or per virtual user (as the [Per-VU Iterations](/using-k6/scenarios/executors/per-vu-iterations)).
```

Comment:
Calculate or configure?

`You can either configure`

Reply:
I think `calculate` makes more sense in this case as we're not talking about the configuration per se, but rather how they are measured/tallied. It's a fine balance between being too technical/going into implementation details, or keeping it high-level and conceptual.